uber/site_sections/preregistration.py

```diff
@@ -67,7 +67,11 @@ def unpaid_preregs(self):
     def paid_preregs(self):
         return cherrypy.session.setdefault('paid_preregs', [])
 
```

Comment:
and btw, I think that in all production versions, this just happened to work OK by happy coincidence.

in the future, we might have multiple ways to access ubersystem (i.e. like what we're almost moving towards with making uber accessible at-even both on-site and on the internet), we will need this to function correctly no matter how your request came in.

extra sidenote: I only discovered this when I had a breakpoint on HTTPRedirect::__init__() and it was going off at server start.  originally, I thought maybe the prereg form was doing some kind of javascript automated check, but it happened even if I closed all my browser windows.  was pretty surprised to find that it was being created as a global object (I swear man python every day I learn something totally crazy about that language, it makes sense though)